external-crates/move/move-vm/profiler/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "move-vm-profiler"
```

Comment:
Not relevant for this PR exactly, but we really should finish flattening all of these crates. We discussed doing it a long time ago but didn't do it. 
In short, any crate should live in `external-crates/move/<package-name-spine-case>`. But we often do `external-crates/move/<package>/<name>` if that makes sense

Reply:
Makes sense but what's the benefit?

Reply:
It makes the repo/codebase more transparent with less insider knowledge. And it simplifies all of the pathing in Cargo.toml. 

There is no real "difference", other than not having to play the game of find the crate